src/controllers/handlers/issue_handler.py

```diff
@@ -0,0 +1,42 @@
+"""This file contains Issue Handler Class"""
+
+import logging
+import shortuuid
+from src.config.regex_pattern import RegexPatterns
+from src.config.sqlite_queries import UserQueries, CreateTable
+from src.config.display_menu import PromptMessage
+from src.database.database_access import DatabaseAccess
+from src.utils.pretty_print import pretty_print
+from src.utils.validate import pattern_validator
+
+logger = logging.getLogger(__name__)
+
+
+def view_issue():
+    """Showing all the Raised Issues"""
+    dao = DatabaseAccess()
+    res_data = dao.execute_returning_query(UserQueries.GET_ALL_ISSUES)
+
+    if len(res_data) == 0:
+        logger.error("No issues Found")
+        print(PromptMessage.NOTHING_FOUND.format("Issues"))
+        return
+
+    headers = ["issue_id", "issue_text", "raised_by"]
```

Comment:
This should be constant in upper case.

Reply:
Resolve this feedback by creating a class with headers constants